cluster/aws/config-default.sh

```diff
@@ -36,12 +36,20 @@ MINION_SCOPES=""
 POLL_SLEEP_INTERVAL=3
 PORTAL_NET="10.0.0.0/16"
 
-# Optional: Install node monitoring.
-ENABLE_NODE_MONITORING=true
-
 # Optional: Install node logging
-ENABLE_NODE_LOGGING=true
+ENABLE_NODE_LOGGING=false
 LOGGING_DESTINATION=elasticsearch # options: elasticsearch, gcp
+
+# Optional: When set to true, Elasticsearch and Kibana will be setup as part of the cluster bring up.
+ENABLE_CLUSTER_LOGGING=false
+if [[ "${ENABLE_NODE_LOGGING-}" == "true" ]]; then
+  if [[ "${LOGGING_DESTINATION-}" == "elasticsearch" ]]; then
+    if [[ "${ENABLE_CLUSTER_LOGGING-}" == "true" ]]; then
```

Comment:
I am sure I just mis-understood you. Shall I remove two of the three tests, or just set the variable at the top-level like I did originally?